test/extensions/filters/network/common/redis/client_impl_test.cc

```diff
@@ -64,6 +64,11 @@ class RedisClientImplTest : public testing::Test, public Common::Redis::DecoderF
     upstream_connection_ = new NiceMock<Network::MockClientConnection>();
     Upstream::MockHost::MockCreateConnectionData conn_info;
     conn_info.connection_ = upstream_connection_;
+
+    // Create timers in order they are created in client_impl.cc
+    connect_or_op_timer_ = {new Event::MockTimer(&dispatcher_)};
```

Comment:
nit: now braces needed here and next line.

Reply:
ok